repositories/PageRepository.js

```diff
@@ -95,6 +97,20 @@ function getRoutingDestinations(pageId) {
   return db.transaction(trx => getAvailableRoutingDestinations(trx, pageId));
 }
 
+function copyPage(id, position) {
```

Comment:
Rename to `duplicatePage` so that our nomenclature is consistent?

Reply:
Great use of the word "nomenclature".